x509_build/install.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
The main reason I used bash is that I don't have a mental model of which constructs are bash-specific and which are also present in sh, which makes it risky that I'll accidentally use a bash construct in sh if I habitually use sh.  What systems lack Bash?  I was under the impression that basically all GNU systems have Bash; are you talking about non-GNU things like Busybox?